contents/docs/feature-flags/snippets/bootstrapping-intro.mdx

```diff
@@ -1,3 +1,3 @@
 Since there is a delay between initializing PostHog and fetching feature flags, feature flags are not always available immediately. This makes them unusable if you want to do something like redirecting a user to a different page based on a feature flag.
 
-To have your feature flags available immediately, you can initialize PostHog with precomputed values until it has had a chance to fetch them. This is called bootstrapping.
\ No newline at end of file
+To have your feature flags available immediately, you can initialize PostHog with precomputed values until it has had a chance to fetch them. This is called bootstrapping. After the SDK is able to fetch feature flags from PostHog, it will use those flag values instead of bootstrapped flag values.
```

Comment:
```suggestion
To have your feature flags available immediately, you can initialize PostHog with precomputed values until it has had a chance to fetch them. This is called bootstrapping. After the SDK fetches feature flags from PostHog, it will use those flag values instead of bootstrapped ones.
```

Reply:
Thank you Ivan 😃